src/util/buildBlankReplacer.js

```diff
@@ -0,0 +1,6 @@
+export default counter => {
+  return () => `<span class='blank'>
+                  <span>${++counter.val}</span>
```

Comment:
Why switching from postfix to prefix increment?

Reply:
Because of the value now starting at 0 instead of 1. I made the change just because of a situation where there are HTML elements inside of the question that have no `@blank` in them and I am relying on the correct count to render inputs and allow submission.

Reply:
Oh, I missed that part.